.bloodhound.yml

```diff
@@ -4,6 +4,8 @@ k8s_versions:
   - 1.28.7
   - 1.29.6
   - 1.30.1
```

Comment:
Can you remove this line, we don't need to validate against `1.30.1` anymore.

Reply:
Ok removed as suggested , Martin.